From 4f332329790686d30d3bab19260ce5909bf80f52 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 9 Feb 2018 19:21:13 +0800
Subject: [PATCH] Revert "Revert "ARM: rockchip_defconfig: enable
 CONFIG_IKCONFIG""

This reverts commit cbd1d90f723ab029e3326c0e085ea050691294ec.

8588d88f1a37 ("ANDROID: android-base.cfg: add CONFIG_IKCONFIG option")

This adds CONFIG_IKCONFIG and CONFIG_IKCONFIG_PROC options, which are a
requirement for the O release.

Change-Id: I438ade881219034618afb4cd5174bf750a79d319
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 28eb935ba761..0ca0c6bb75c6 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -8,6 +8,8 @@ CONFIG_TASKSTATS=y
 CONFIG_TASK_DELAY_ACCT=y
 CONFIG_TASK_XACCT=y
 CONFIG_TASK_IO_ACCOUNTING=y
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=19
 CONFIG_CGROUPS=y
 CONFIG_CGROUP_FREEZER=y
-- 
2.35.3

